staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -344,6 +344,11 @@ func (p *jsonPatcher) applyPatchToCurrentObject(requestContext context.Context,
 		}
 	}
 
+	if p.options == nil {
+		// Provide a more informative error for the crash that would
+		// happen on the next line
+		panic("PatchOptions required but not provided")
+	}
```

Comment:
Have you seen that panic? Clearly it shouldn't happen. I'm guessing that the added options below?